### PR TITLE
feat: uosc support

### PR DESCRIPTION
**Pull request by Lichthagel**
Adds support for [uosc](https://github.com/tomasklaen/uosc), a "Feature-rich minimalist proximity-based UI for MPV player."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,9 +40,40 @@
 2. Choose an accent from the flavor directory
 3. Copy the contents of the accent directory into `~/.config/mpv`.
 
+## uosc support
```
Comment (sgoudham) [inline README.md]: I don't think we should clutter the main README by copying the template again, maybe put this and the related files into its own `uosc` sub-directory?

**Review by sgoudham — CHANGES_REQUESTED** [review]

File: justfile [hunk 2]
```diff
@@ -43,3 +43,47 @@ clean:
...
+
+# ---- uosc ----
+
+# Variables
+output_uosc := "themes-uosc"
+template_path_uosc := "uosc.tera"
+
+# Create the output directory
+# TODO remove when PR #5 is merged
+setup_uosc:
```
Comment (Anomalocaridid) [inline justfile]: #5 has now been merged. Could you please remove `setup_uosc`?

File: justfile [hunk 3]
```diff
@@ -43,3 +43,47 @@ clean:
...
+    cp -vr {{output_uosc}}/latte/* {{output_uosc}}/frappe
+    cp -vr {{output_uosc}}/latte/* {{output_uosc}}/macchiato
+    cp -vr {{output_uosc}}/latte/* {{output_uosc}}/mocha
+
+# Remove all files in the output directory
+clean_uosc:
+    rm -rfv {{output_uosc}}
+
+# Generate all four flavors
+all_uosc: setup_uosc
```
Comment (Anomalocaridid) [inline justfile]: It might be a good idea to have `clean_uosc` and `all_uosc` be dependencies of `clean` and `all`.

**Review by Anomalocaridid — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: justfile (hunk 3 above)
Comment (Lichthagel) [reply justfile]: Well, my idea was to keep the main theme separate.

Now, with a separate justfile in a subdirectory I added `just uosc/{clean,all}` in front. Idk if there is a cleaner way.

Comment (Anomalocaridid) [reply justfile]: Understandable. With the subdirectory and separate justfile, it should be alright how it currently is.

**Review by Anomalocaridid — APPROVED** [review]
LGTM!

**Review by sgoudham — CHANGES_REQUESTED** [review]
Thanks for moving it out to a separate directory, I've noticed that the volume bar on the Latte screenshot has low contrast:

image

I believe we should be using `text` for accented backgrounds on Latte?

**Comment by Lichthagel** [comment]
> I've noticed that the volume bar on the Latte screenshot has low contrast:

Well contrast wouldn't be a problem, if I didn't use mocha's mauve in all screenshots 🤦🏻 .

latte

**Review by sgoudham — APPROVED** [review]

**Comment by Anomalocaridid** [comment]
Thank you!

**Pull request merged**